src/uct/base/uct_pd.c

```diff
@@ -297,7 +297,7 @@ ucs_status_t uct_iface_config_read(const char *tl_name, const char *env_prefix,
                                    const char *filename,
                                    uct_iface_config_t **config_p)
 {
-    uct_config_bundle_t *bundle;
+    uct_config_bundle_t *bundle = NULL;
```

Comment:
gcc started to complain about un-initialized variable, because it could not tell that 'status' var controls the usage of 'bundle'. probably the inlining/optimization changed.